lib/new_inject.rb

```diff
@@ -0,0 +1,31 @@
+class Array
+  def new_inject(*args, &block)
```

Comment:
Unused method argument - `block`. If it's necessary, use `_` or `_block` as an argument name to indicate that it won't be used.<br>Assignment Branch Condition size for new_inject is too high. [27.96/15]